nvflare/lighter/project.yml

```diff
@@ -76,7 +76,7 @@ builders:
       components:
         server:
           - id: job_scheduler  # This id is reserved by system.  Do not change it.
-            path: nvflare.apis.impl.job_scheduler.DefaultJobScheduler
+            name: DefaultJobScheduler
```

Comment:
I still prefer our current decision to use path in stead of name for all components and classes.  Not sure if we are going back to use name.

Reply:
no problem, then I will change to path